gravitee-apim-console-webui/src/management/api/endpoints-v4/endpoint-group/create/api-endpoint-group-create.component.ts

```diff
@@ -84,6 +85,11 @@ export class ApiEndpointGroupCreateComponent implements OnInit {
           const apiV4 = api as ApiV4;
           this.generalForm.get('name').addValidators([isUnique(apiV4.endpointGroups.map((group) => group.name))]);
           this.apiType = apiV4.type;
+          this.requiredQos = apiV4.listeners
+            .map((listener) => listener.entrypoints)
```

Comment:
Why not just `flatMap` for entrypoints and then for qos?

Reply:
Fixed, forgot about `flatMap`